src/map/atcommand.cpp

```diff
@@ -8307,14 +8301,12 @@ ACMD_FUNC(invite)
 		return -1;
 	}
 
-	if(did == 0) {
+	if(did == 0 || !duel_exist(did) ) {
 		clif_displaymessage(fd, msg_txt(sd,350)); // "Duel: @invite without @duel."
 		return 0;
 	}
 
-	if(duel_list[did].max_players_limit > 0 &&
-		duel_list[did].members_count >= duel_list[did].max_players_limit) {
-
+	if( !duel_check_player_limit( duel_GetDuelId(did) ) ){
```

Comment:
totally against of mixing UpperCaseAndLower_CaseTogether. Lets stick to only one lower with "_". 😃